packages/core/src/lib/util.ts

```diff
@@ -146,6 +157,14 @@ export function generatePackageId(host: string): string | null {
     if (part.trim().length === 0) {
       continue;
     }
+
+    // Package names cannot contain Java keywords. The recommendation is adding an '_' before the
+    // keyword. See https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html.
+    if (JAVA_KEYWORDS.indexOf(part) >= 0) {
```

Comment:
Can't we just use `JAVA_KEYWORDS.includes(part)`?

Reply:
Oh, silly me... I remembered there was a handy method in JS Array, but couldn't remember the name. I checked the docs, but couldn't find what I was looking for (I think I was looking for something like `contains`), and missed `includes`... I'll change that.

Reply:
done